modules/osdk-hh-helm-reconciler.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-hybrid-helm.adoc
+
+:_content-type: CONCEPT
+[id="osdk-hh-helm-reconciler_{context}"]
+= Custom Helm reconciler configurations using provided library APIs
+
+A disadvantage of existing Helm-based Operators is the inability to configure the Helm reconciler, because it is abstracted from users. For a Helm-based Operator to reach the Seamless Upgrades capability (level II and later) that reuses an already existing Helm chart, a hybrid between the Go and Helm Operator types adds value.
+
+The APIs provided in the link:https://github.com/operator-framework/helm-operator-plugins[`helm-operator-plugins`] library allow Operator authors to do the following:
+
+* Customize value mapping based on cluster state
+* Execute code in specific events by configuring the reconciler's event recorder
+* Customize the reconciler's logger
+* Setup `Install`, `Upgrade`, and `Uninstall` annotations to enable Helm's actions to be configured based on the annotations found in custom resources watched by the reconciler
+* Configure the reconciler to run with `Pre` and `Post` hooks
+
+The above configurations to the reconciler can be done in the `main.go` file:
```

Comment:
If you add the _configurations to the reconciler_ phrase above, s/configurations to the reconciler/configurations